libs/packages/components/src/lib/search-result-list/search-result-list.component.ts

```diff
@@ -36,14 +41,17 @@ export class SdsSearchResultListComponent  {
    */
   @Input() divider = true;
 
-  constructor(private _location: Location) 
-  {}
+  constructor(private _location: Location) {}
+
+  ngOnInit() {
+    this.initialLoad = true;
+  }
```

Comment:
Small suggestion - would initializing initialLoad as true instead of false in line 22 allow you to remove the onInit method here?

Reply:
I need that way because if I do true on line 22 initial set() is executing first at that time it will be false.